jsapp/js/components/table.es6

```diff
@@ -442,8 +442,12 @@ export class DataTable extends React.Component {
                   value={filter ? filter.value : ''}>
             <option value=''>Show All</option>
             {choices.filter(c => c.list_name === col.question.select_from_list_name).map((item, n) => {
+              let displayLabel = t('Unlabelled');
+              if (item.label) {
+                displayLabel = item.label[0];
+              }
               return (
-                <option value={item.name} key={n}>{item.label[0]}</option>
+                <option value={item.name} key={n}>{displayLabel}</option>
```

Comment:
Let's show the `item.name` if there's no `item.label`. You can see that behavior already in the form builder, e.g. if you remove an option/choice from one version to the next:
![image](https://user-images.githubusercontent.com/2085013/49776714-28ed4880-fccb-11e8-9d50-4589657d6403.png)

Reply:
I also added fallback to `$autoname` :)